phpcs.xml.dist

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<ruleset>
+    <arg name="basepath" value="." />
+    <arg name="extensions" value="php" />
+    <arg name="parallel" value="80" />
+    <arg name="cache" value=".phpcs-cache" />
+    <arg name="colors" />
+
+    <!-- Ignore warnings, show progress of the run and show sniff names -->
+    <arg value="nps" />
+
+    <file>src</file>
+    <file>tests</file>
+
+    <rule ref="Doctrine">
```

Comment:
cdn77 plssss :cry:

Reply:
tam jsou neprijemny indenty

Reply:
Jaký indenty? A to je celkem jedno, je na hovno dělat firemní věci a používat na to jiný CS ...

Reply:
to neni firemni vec, ja si to spichnul o vikendu nonbillable, protoze me ten puvodni uz iritoval

assigmenty nejsou zarovnany pod sebe, tak to na vecech mimo nepouzivam no

Reply:
přijde mi fakt debilní muset přepínat mezi více cs no

Reply:
mozna bych to po releasu cdn mohl prepnout a excludnout nas exclude 🤔 aspon nez to protlacim do doctrine